src/main/kotlin/com/wire/apps/polls/dto/app/Factories.kt

```diff
@@ -0,0 +1,85 @@
+package com.wire.apps.polls.dto.app
+
+import com.wire.integrations.jvm.model.QualifiedId
+import com.wire.integrations.jvm.model.WireMessage
+import java.util.UUID
+
+/**
+ * Creates message for poll.
+ */
+fun newPoll(
+    conversationId: QualifiedId,
+//    id: String,
+    body: String,
+    buttons: List<WireMessage.Composite.Button>
+//    mentions: List<WireMessage.Text.Mention> = emptyList()
+): WireMessage.Composite =
+    WireMessage.Composite.create(
+        conversationId = conversationId,
+        text = body,
+        buttonList = buttons
+    )
+
+/**
+ * Creates message for vote confirmation.
+ */
+fun confirmVote(
+    pollId: String,
+    conversationId: QualifiedId,
+    offset: Int
+): WireMessage.ButtonActionConfirmation {
+    return WireMessage.ButtonActionConfirmation(
```

Comment:
Here you can simplify the return as the other functions

```suggestion
): WireMessage.ButtonActionConfirmation =
    WireMessage.ButtonActionConfirmation(
```